### ISSUE TITLE
A11y_Azure data studio_Query History_Executed query_Screenreader: Screenreader is not reading the completed information present under the Query history "SELECT 9" Executed query

### ISSUE THREAD

**Issue by Jtith** [microsoft/azuredatastudio#20889]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #SH_SQLAzureDataStudio_Win32_Oct2022; #SQL Azure Data Studio; #WCAG1.3.1; #Screen reader; #NVDA; #JAWS; #Win11; #FTP; #DesktopApp; #Win32; #Element:Tablerow;
### Environment Details:
Application Name: SQL Azure Data Studio
Application Version: 1.40.0- insider
Windows 11 Version: 22621.674
NVDA Version 2020.4
JAWS Version 2021.205.53

### Repro Steps: -

1. Install Azure Data Studio – Insiders
2. Tab till New connection and press enter.
3. Using valid credentials, connect to server.
4. Tab till extensions and press enter.
5. Tab till search box and search "Query History". 
6. Tab till Install and press enter.
7. Tab till new query and press enter
8. Tab till query editor and execute any query.
9. Open Toggle panel using Ctrl+J.
10. Tab till Query History tab and press enter.
11. Turn on NVDA
12. Tab till the executed query in rows below Query history.
13. Check whether the screen reader is announcing the complete information present under the Query history row section.

### Actual:
Screen reader is not reading the completed information present under the Query history "SELECT 9" Executed query row.

### Expected:
Screen reader should read the completed information present under the Query history "SELECT 9" row Executed query row.

Ex: When tab focus goes to SELECT 9 it should announce SELECT 9 sqlTools2019-3|master 10/18/2022, 5:17:56 PM

### User Impact:
Screen reader user will get complete information of Query history data.

### Attachment: -
![Screenreader-is-not-reading-the](https://user-images.githubusercontent.com/100552367/196866858-a228534e-e203-4ff5-9247-22a012dc6bed.gif)
![A11y-Azure-data-studio-Query-His](https://user-images.githubusercontent.com/100552367/196962058-24f358ba-e2e8-471e-9069-c76d67a5e00b.gif)

**Comment by Charles-Gagnon** [comment]
@Jtith This issue has been fixed, please make sure you have at least the `0.5.2` version of the extension installed when verifying.

**Comment by Charles-Gagnon** [comment]
@Jtith I'm closing this since we haven't heard back in a couple of weeks. Please let us know if you're able to verify the issue is fixed.

**Comment by Jtith** [comment]
@Charles-Gagnon Please provide the latest environment and version details to verify the issue
